chart/templates/flower/flower-deployment.yaml

```
@@ -67,6 +67,7 @@ spec:
       tolerations:
 {{ toYaml $tolerations | indent 8 }}
       serviceAccountName: {{ include "flower.serviceAccountName" . }}
+      priorityClassName: {{ .Values.flower.priorityClassName }}
```

I think we should conditionally add these:

```suggestion
      {{- if .Values.flower.priorityClassName }}
      priorityClassName: {{ .Values.flower.priorityClassName }}
      {{- end }}
```

---

Changed.